packages/react/src/exampleTypes/ResultTable/Header.js

```diff
@@ -81,136 +90,138 @@ let Header = ({
     F.flip(filtering)()
   }
   let Label = label
+  let { className, style, ...headerCellProps } =
+    fieldSchema.headerCellProps ?? {}
   return (
     <HeaderCell
       className={`${isStickyColumn ? 'sticky-column-header' : ''}
-        ${_.get('hasValue', filterNode) ? 'active-filter' : ''}`}
+        ${_.get('hasValue', filterNode) ? 'active-filter' : ''} ${className}`}
       style={{
         cursor: hideMenu ? 'default' : 'pointer',
         left: isStickyColumn ? 0 : '',
+        ...style,
       }}
+      {...headerCellProps}
     >
-      <span>
-        {_.isFunction(label) ? <Label /> : label}{' '}
-        {field === node.sortField && (
-          <Icon
-            icon={node.sortDir === 'asc' ? 'SortAscending' : 'SortDescending'}
-          />
+      {_.isFunction(label) ? <Label /> : label}{' '}
+      {field === node.sortField && (
+        <Icon
+          icon={node.sortDir === 'asc' ? 'SortAscending' : 'SortDescending'}
+        />
+      )}
+      <Popover
+        trigger={hideMenu ? null : <Icon icon="TableColumnMenu" />}
+        position={`bottom ${isLastColumn ? 'right' : 'center'}`}
+        closeOnPopoverClick={false}
+        style={popoverStyle}
+      >
+        {!disableSort && (
+          <DropdownItem
+            onClick={() => {
+              mutate({ sortField, sortDir: 'asc' })
+            }}
+          >
+            <Icon icon="SortAscending" />
+            Sort Ascending
+          </DropdownItem>
         )}
-        <Popover
-          trigger={hideMenu ? null : <Icon icon="TableColumnMenu" />}
-          position={`bottom ${isLastColumn ? 'right' : 'center'}`}
-          closeOnPopoverClick={false}
-          style={popoverStyle}
-        >
-          {!disableSort && (
-            <DropdownItem
-              onClick={() => {
-                mutate({ sortField, sortDir: 'asc' })
-              }}
-            >
-              <Icon icon="SortAscending" />
-              Sort Ascending
-            </DropdownItem>
-          )}
-          {!disableSort && (
-            <DropdownItem
-              onClick={() => {
-                mutate({ sortField, sortDir: 'desc' })
-              }}
-            >
-              <Icon icon="SortDescending" />
-              Sort Descending
-            </DropdownItem>
-          )}
+        {!disableSort && (
```

Comment:
A request to consolidate the components under a single guard since we're using `{!disableSort &&` twice.